### PR TITLE
Extract On-Call docs out of wiki

### PR DESCRIPTION
**Pull request by spiffxp**
This came up during the most recent community meeting as step 1 of proposing changes to on-call policies.

No content changes aside from formatting/links.  Moved into `docs/devel/on-call-*.md`

Someone with edit rights to the wiki will need to remove these pages:
- https://github.com/kubernetes/kubernetes/wiki/Kubernetes-on-call-rotations
- https://github.com/kubernetes/kubernetes/wiki/Github-and-Build-Cop-Rotation
- https://github.com/kubernetes/kubernetes/wiki/User-Support-Rotation

This is my first docs PR so I'm not sure if this needs to be cherry-picked into release branches?

/cc @k8s-oncall @kubernetes/goog-testing @kubernetes/sig-testing

### CODE REVIEW

File: docs/devel/on-call-build-cop.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    * There is a related concept of "Github teams" which allow you to @ mention a set of people; feel free to @ mention a Github team if you wish, but this is not a substitute for adding a team/* label, which is required
+      * [Google teams](https://github.com/orgs/kubernetes/teams?utf8=%E2%9C%93&query=goog-)
+      * [Redhat teams](https://github.com/orgs/kubernetes/teams?utf8=%E2%9C%93&query=rh-)
+      * [SIGs](https://github.com/orgs/kubernetes/teams?utf8=%E2%9C%93&query=sig-)
+    * If the issue is reporting broken builds, broken e2e tests, or other obvious P0 issues, label the issue with priority/P0 and assign it to someone. This is the only situation in which you should add a priority/* label
+      * non-P0 issues do not need a reviewer assigned initially
+    * Assign any issues related to Vagrant to @derekwaynecarr (and @mention him in the issue)
+  * All incoming PRs should be assigned a reviewer.
+    * unless it is a WIP (Work in Progress), RFC (Request for Comments), or design proposal.
```
Comment (jayunit100) [inline docs/devel/on-call-build-cop.md]: these nested bullets seem a little confusing ?

Comment (spiffxp) [inline docs/devel/on-call-build-cop.md]: I'd rather defer content edits to another PR, and keep this scoped to transfer of content from one place to another.  This was pretty much copy-pasta out of the wiki

**Comment by ihmccreery** [comment]
LGTM; sorry for the delay.

**Comment by spiffxp** [comment]
@ihmccreery thanks!  pages remain in the wiki, should I open another issue to get those deleted?  I don't seem to have write access

**Comment by ihmccreery** [comment]
Thanks @spiffxp; open an issue with links, and I will delete them ASAP to avoid drift.

**Comment by bgrant0607** [comment]
In the future, please leave placeholders to redirect people to the new location. All existing links are now broken.

**Comment by ihmccreery** [comment]
@bgrant0607 Oops, sorry.  Fixed.

**Pull request merged**